js/modules/aioseop_module.js

```diff
@@ -1,345 +1,606 @@
-		return false;
-	});
-	jQuery("div#aiosp_sitemap_addl_pages_metabox").delegate("a.aiosp_delete_url", "click", function(e) {
-		e.preventDefault();
-		aioseop_handle_post_url('aioseop_ajax_delete_url', 'sitemap_addl_pages', jQuery(this).attr("title") );
-		return false;
-	});
-	jQuery("div#aiosp_video_sitemap_addl_pages_metabox").delegate("a.aiosp_delete_url", "click", function(e) {
-		e.preventDefault();
-		aioseop_handle_post_url('aioseop_ajax_delete_url', 'video_sitemap_addl_pages', jQuery(this).attr("title") );
-		return false;
-	});
-	jQuery("div#aiosp_opengraph_scan_header").delegate("input[name='aiosp_opengraph_scan_header']", "click", function(e) {
-		e.preventDefault();
-		aioseop_handle_post_url('aioseop_ajax_scan_header', 'opengraph_scan_header', jQuery('div#aiosp_opengraph_scan_header').serialize() );
-		return false;
-	});
-	jQuery( 'input[name="aiosp_sitemap_posttypes[]"][value="all"], input[name="aiosp_video_sitemap_posttypes[]"][value="all"], input[name="aiosp_sitemap_taxonomies[]"][value="all"], input[name="aiosp_video_sitemap_taxonomies[]"][value="all"]' ).click(function () {
-		jQuery(this).parents('div:eq(0)').find(':checkbox').prop('checked', this.checked);
-    });
-	jQuery( 'input[name="aiosp_sitemap_posttypes[]"][value!="all"], input[name="aiosp_video_sitemap_posttypes[]"][value!="all"], input[name="aiosp_sitemap_taxonomies[]"][value!="all"], input[name="aiosp_video_sitemap_taxonomies[]"][value!="all"]' ).click(function () {
-		if ( !this.checked )
-			jQuery(this).parents('div:eq(0)').find('input[value="all"]:checkbox').prop('checked', this.checked);
-    });
-
-	jQuery(".aioseop_tab:not(:first)").hide();
-    jQuery(".aioseop_tab:first").show();
-    jQuery("a.aioseop_header_tab").click(function(){
-            var stringref = jQuery(this).attr("href").split('#')[1];
-            jQuery('.aioseop_tab:not(#'+stringref+')').hide('slow');
-            jQuery('.aioseop_tab#' + stringref).show('slow');
-            jQuery('.aioseop_header_tab[href!="#'+stringref+'"]').removeClass('active');
-            jQuery('.aioseop_header_tab[href="#' + stringref+'"]').addClass('active');
-            return false;
-    });    
-});
+/**
+ * Controls all the styling of the plugin.
+ *
+ * AIOSEOP Updates class.
+ * @author Michael Torbert.
+ * @author Semper Fi Web Design.
+ * @copyright http://semperplugins.com
+ * @version 1.0.0
+ */
+if ( typeof aiosp_data != 'undefined' ) {
+
+	/**
+	 * @since 1.0.0
+	 * @param int $index.
+	 * @param $value
+	 */
+	jQuery.each( aiosp_data, function( index, value ) {
+//		aiosp_data[index] = value.json.replace(/&quot;/g, '"');
+//		aiosp_data[index] = jQuery.parseJSON( value );
+		if ( index == 0 ) {
+			if ( typeof value.condshow == 'undefined' ) {
+				aiosp_data[ index ].condshow = [];
+			}
+		} else {
+			if ( typeof value.condshow != 'undefined' ) {
+				aiosp_data[ 0 ].condshow =

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'aiosp_data' is not in camel case.](https://www.codacy.com/app/mrtorbert/all-in-one-seo-pack/file/2692638761/issues/source?bid=3283401&fileBranchId=3340169#l26)